.github/workflows/ci.yml

```
@@ -0,0 +1,49 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - main
+
+concurrency:
+  group: ${{ github.ref_name }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+env:
+  BUNDLE_WITHOUT: "console:development:journald:libvirt"
```

Hah, I suggested this, but it won't work for katello, as there rubocop is in `development`: https://github.com/Katello/katello/blob/master/katello.gemspec#L70

So, let's do
```suggestion
  BUNDLE_WITHOUT: "console:journald:libvirt"
```